src/main/resources/db/migration/changes/001-create-team-tables.json

```diff
@@ -0,0 +1,105 @@
+{ "databaseChangeLog": [
```

Comment:
Depending on the order in which PRs get merged, this might need to be 002 instead of 001; we'll see!